eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -53,6 +58,23 @@ function GetAdjustedReadmeContent($ReadmeContent, $PackageInfo, $PackageMetadata
   # The $PackageMetadata could be $null if there is no associated metadata entry
   # based on how the metadata CSV is filtered
   $service = $PackageInfo.ServiceDirectory.ToLower()
+  Write-Host "Service name is $service."
+  if ($VsoVariable) {
+    $alreadyPresent = [System.Environment]::GetEnvironmentVariable($VsoVariable)
+
+    if ($alreadyPresent) { 
+      $authorMetadataJson = $alreadyPresent | ConvertFrom-Json
+    }
+  }
+  $author = 'ramya-rao-a'
+  $msauthor = 'ramyar'
```

Comment:
I think fallback order here would look like: 

* Use author metadata from variable... if no metadata
* Read owner info from a file in the repo (`eng/docs-metadata-defaults.json`) or from a variable set in `Language-Settings.ps1` ... we will want different fallback owners for different repositories ... if no file...
* Hardcoded fallback